com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -86,29 +164,80 @@ public Matrix4x4 GetProjectionMatrix(int viewIndex = 0)
         /// Similar to <c>GL.GetGPUProjectionMatrix</c> but queries URP internal state to know if the pipeline is rendering to render texture.
         /// For more info on platform differences regarding camera projection check: https://docs.unity3d.com/Manual/SL-PlatformDifferences.html
         /// </summary>
+        /// <param name="viewIndex"> View index in case of stereo rendering. By default <c>viewIndex</c> is set to 0. </param>
         /// <seealso cref="GL.GetGPUProjectionMatrix(Matrix4x4, bool)"/>
         /// <returns></returns>
         public Matrix4x4 GetGPUProjectionMatrix(int viewIndex = 0)
         {
             return GL.GetGPUProjectionMatrix(GetProjectionMatrix(viewIndex), IsCameraProjectionMatrixFlipped());
         }
 
+        /// <summary>
+        /// The camera component.
+        /// </summary>
         public Camera camera;
+
+        /// <summary>
+        /// The camera render type used for camera stacking.
+        /// <see cref="CameraRenderType"/>
+        /// </summary>
         public CameraRenderType renderType;
+
+        /// <summary>
+        /// Controls the final target texture for a camera. If null camera will resolve rendering to screen.
+        /// </summary>
         public RenderTexture targetTexture;
+
+        /// <summary>
+        /// Render texture settings used to create intermediate camera textures for rendering.
+        /// </summary>
         public RenderTextureDescriptor cameraTargetDescriptor;
         internal Rect pixelRect;
         internal int pixelWidth;
         internal int pixelHeight;
         internal float aspectRatio;
+
+        /// <summary>
+        /// Render scale to apply when creating camera textures.
+        /// </summary>
         public float renderScale;
+
+        /// <summary>
+        /// True if this camera should clear depth buffer. This setting only applies to cameras of type <c>CameraRenderType.Overlay</c>
+        /// <seealso cref="CameraRenderType"/>
+        /// </summary>
         public bool clearDepth;
+
+        /// <summary>
+        /// The camera type.
+        /// <seealso cref="UnityEngine.CameraType"/>
+        /// </summary>
         public CameraType cameraType;
+
+        /// <summary>
+        /// True if this camera is drawing to a viewports that maps to the entire screen.
+        /// </summary>
         public bool isDefaultViewport;
+
+        /// <summary>
+        /// True if this camera should render to high definition color targets
```

Comment:
Nit: I think "high definition" is a bit misleading, as it could imply reference to HDRP or high resolution. Should we just explicitly say "high dynamic range color targets"?

Reply:
fixed